### PR TITLE
bibtext and reference.bib updated for 'Correction for multiple comparison using FDR' in multi_comp.py

### LINKED ISSUE #8930: use bibtex in multi_comp.py
convert references in `mne/stats/multi_comp.py` to use footcite / footbibliography

### PR DESCRIPTION
**Pull request by FinLouarn**
#### Reference issue
Fixes #8930 .

#### What does this implement/fix?
updated bibtext and references.bib for 'Correction for multiple comparison using FDR' in multi_comp.py

#### Additional information
The html doc have been built properly locally

### CODE REVIEW

**Review by drammock — APPROVED** [review]

**Comment by drammock** [comment]
This one has a merge conflict...  probably you didn't update `main` from upstream before creating the new branch.  The solution is to `git fetch upstream` and then `git rebase upstream/main`.  Then, type `git mergetool` to open a text editor.  You will see that you and someone else both added new BibTeX references to the end of the file.  You need to keep them both, and remove any of the `<<<< HEAD` and `=====` type of lines, save and close the file, and then run `git rebase --continue`.

**Review by drammock — APPROVED** [review]

**Review by agramfort — APPROVED** [review]
LGTM

File: doc/references.bib [hunk 1]
```diff
@@ -2023,4 +2023,19 @@ @Article{Kappenman2021
   publisher = {Elsevier {BV}},
 }
 
+@article{GENOVESE2002870,
```
Comment (larsoner) [inline doc/references.bib:2026]: Can you try to match the format for other entries in this file? Probably `GenoveseEtAl2002`?

Comment (sappelhoff) [reply doc/references.bib:2026]: I agree it'd be nice to stay consistent with the other references here.

File: doc/references.bib [hunk 2]
```diff
@@ -2023,4 +2023,19 @@ @Article{Kappenman2021
...
+keywords = {functional neuroimaging, false discovery rate, multiple testing, Bonferroni correction},
+abstract = {Findingobjective and effective thresholds for voxelwise statistics derived from neuroimaging data has been a long-standing problem. With at least one test performed for every voxel in an image, some correction of the thresholds is needed to control the error rates, but standard procedures for multiple hypothesis testing (e.g., Bonferroni) tend to not be sensitive enough to be useful in this context. This paper introduces to the neuroscience literature statistical procedures for controlling the false discovery rate (FDR). Recent theoretical work in statistics suggests that FDR-controlling procedures will be effective for the analysis of neuroimaging data. These procedures operate simultaneously on all voxelwise test statistics to determine which tests should be considered statistically significant. The innovation of the procedures is that they control the expected proportion of the rejected hypotheses that are falsely rejected. We demonstrate this approach using both simulations and functional magnetic resonance imaging data from two simple experiments.}
+}
```
Comment (britta-wstnr) [inline doc/references.bib:2039]: The abstract could be deleted to prevent cluttering in the references file. Thoughts, @larsoner @sappelhoff ?

Comment (sappelhoff) [reply doc/references.bib:2039]: agreed

File: doc/references.bib (hunk 1 above)
Comment (sappelhoff) [reply doc/references.bib:2026]:
```suggestion
@article{GenoveseEtAl2002,
```

File: doc/references.bib (hunk 2 above)
Comment (sappelhoff) [inline doc/references.bib:2039]:
```suggestion
keywords = {functional neuroimaging, false discovery rate, multiple testing, Bonferroni correction}
}
```

File: mne/stats/multi_comp.py [hunk 3]
```diff
@@ -17,7 +17,7 @@ def _ecdf(x):
 def fdr_correction(pvals, alpha=0.05, method='indep'):
     """P-value correction with False Discovery Rate (FDR).
 
-    Correction for multiple comparison using FDR [1]_.
+    Correction for multiple comparison using FDR :footcite:`GENOVESE2002870`.
```
Comment (sappelhoff) [inline mne/stats/multi_comp.py:20]:
```suggestion
    Correction for multiple comparison using FDR :footcite:`GenoveseEtAl2002`.
```

**Review by sappelhoff — COMMENTED** [review]
I turned the two remaining comments into suggestions which I'll accept to make this ready for merge.

be sure to "git fetch" and "git merge --ff-only" before continuing to work on this

**Review by sappelhoff — APPROVED** [review]

**Comment by sappelhoff** [comment]
thanks @FinLouarn

**Pull request merged**